django/contrib/contenttypes/migrations/0001_initial.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
                 ('model', models.CharField(max_length=100, verbose_name='python model class name')),
             ],
             options={
-                'ordering': ('name',),
+                'ordering': ['name'],
```

Comment:
This change is unrelated, I'm going to revert it.

Reply:
No problem, but do you mind explaining a bit? Above you requested changes to the migrations as well.